app/controllers/api/v1/cars_controller.rb

```diff
@@ -1,12 +1,40 @@
 class Api::V1::CarsController < ApplicationController
   def index
     @cars = Car.all
-    render json: @cars
+    if @cars.empty?
+      render json: { message: 'No cars found' }
+    else
+      render json: @cars
+    end
   end
 
-  def new; end
+  def show
+    @car = Car.find(params[:id])
+    render json: @car
+  end
```

Comment:
Status codes could be used here as well and if statement as well to handle cases where the car is not found.

Reply:
alrigh. let me do